### PR TITLE
docs

### PR DESCRIPTION
**Pull request by basz**
@svycka Does this make sense?

### CODE REVIEW

**Comment by svycka** [comment]
don't think so.. PR https://github.com/juriansluiman/SlmQueue/pull/201 affects only users who does not use zend-mvc and anyway they can still get eventmanager with this https://github.com/juriansluiman/SlmQueue/blob/master/src/Worker/AbstractWorker.php#L79 it is not a shared event manager but it is eventmanager anyway...

**Comment by basz** [comment]
so adding something like

`or you should attach listeners to the manager from your Worker::getEventManager()`

would be more accurate?

**Comment by basz** [comment]
or should we remove this doc at all and suggest `Worker::getEventManager`? https://github.com/basz/SlmQueue/blob/93d194af791b22ea8864dfc3f9665bfca02d9661/docs/6.Events.md#using-the-shared-event-manager

**Comment by svycka** [comment]
I think both are valid not sure with one is more accurate. but I would go with Worker::getEventManager since this is valid both ways.

**Comment by svycka** [comment]
@basz I didn't now we have this in docs but I guess this does not change anything since this example is for  zend-mvc and still working. But I guess to be more clear and safe we could get not shared event manager but event manager from worker because who knows how it is created maybe someone is using different factory and this way using custom event manager... so using event manager from worker always will be safe..

**Comment by svycka** [comment]
I think that in next version maybe we could get ride of shared event manager at all? also not sure if this is important but with zend-mvc Worker::getEventManager will return SharedEventManager but without it will return just EventManager. Is this important and should return the same in both cases?

**Comment by svycka** [comment]
oh I was wrong..
Both has EventManager the difference is that EventManager from zend-mvc is created with attached shared event manager I guess this is the only difference. But anyway if event will be attached to event manager from Worker:getEventManager then there is no big difference if we will not use SharedEventManager and remove from docs or update docs to use event manager from Worker::getEventManager.

**Comment by basz** [comment]
> I think that in next version maybe we could get ride of shared event manager at all

me too

**Comment by svycka** [comment]
if event manager is important in this module maybe would be good idea to introduce container key for eventmanager would be easier to add it.

File: docs/6.Events.md [hunk 1]
```diff
@@ -467,6 +467,32 @@ class Module
 }
 ```
 
+Note: since v1.0.1 we have decoupled from zendframework/zend-mvc and as such the event manager isn't reliably available in the service container. If you are not using zend-mvc and still rely on this feature you should inject a shared event manager into your container yourself.
```
Comment (roelvanduijnhoven) [inline docs/6.Events.md]: I would rephrase as:

> Note: since v1.0.1 we have decoupled from zendframework/zend-mvc and as such the shared event manager isn't available in the service container. If you are not using zend-mvc you should not use the shared event manager.

**Comment by roelvanduijnhoven** [comment]
@basz And went ahead and merged the changes in master. Thanks!
